tests/fsdiff/test_contentdiff.py

```
@@ -136,20 +138,20 @@ def test_json_diff_read_error_fallback(self):
         new_entry = make_entry("/new.json")
         new_entry.file_type_info = info
 
-        # Mock json.load to raise ValueError (simulating corrupt JSON)
-        # We need to mock open as well to return something valid-ish for the fallback text read
-        with patch("json.load", side_effect=ValueError("Bad JSON")):
-            with patch("builtins.open", mock_open(read_data="bad data")):
-                diff = self.manager.generate_content_diff(
-                    Path("/old.json"), Path("/new.json"), old_entry, new_entry
-                )
+        with patch("pathlib.Path.exists", autospec=True) as mock_exists:
```

Ruff (0.14.8)

Local variable `mock_exists` is assigned to but never used

Remove assignment to unused variable `mock_exists`

(F841)